modin/numpy/__init__.py

```diff
@@ -0,0 +1,126 @@
+# Licensed to Modin Development Team under one or more contributor license agreements.
+# See the NOTICE file distributed with this work for additional information regarding
+# copyright ownership.  The Modin Development Team licenses this file to you under the
+# Apache License, Version 2.0 (the "License"); you may not use this file except in
+# compliance with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under
+# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific language
+# governing permissions and limitations under the License.
+
+from .arr import array
+
+from .array_creation import (
+    zeros_like,
+    ones_like,
+)
+
+from .array_shaping import (
+    ravel,
+    shape,
+    transpose,
+)
+
+from .math import (
+    absolute,
+    abs,
+    add,
+    divide,
+    float_power,
+    floor_divide,
+    power,
+    prod,
+    multiply,
+    remainder,
+    mod,
+    subtract,
+    sum,
+    true_divide,
+    mean,
+    maximum,
+    amax,
+    max,
+    minimum,
+    amin,
+    min,
+)
+
+from .constants import (
+    Inf,
+    Infinity,
+    NAN,
+    NINF,
+    NZERO,
+    NaN,
+    PINF,
+    PZERO,
+    e,
+    euler_gamma,
+    inf,
+    infty,
+    nan,
+    newaxis,
+    pi,
+)
+
+
+def where(condition, x=None, y=None):
+    if condition:
```

Comment:
This seems incorrect, since `where` is supposed to check the `condition` array element-wise. Even if the argument here were a modin.numpy array that has the `where` attribute, this check happens too early for that to matter.

```python
>>> import numpy
>>> numpy.where([1, 0], [1, 1], [2, 2])
array([1, 2])
>>> import modin.numpy as np
>>> np.where([1,0], [1,1],[2,2])
[1, 1]
```

Reply:
Good point - fixed!